src/dbnode/encoding/m3tsz/float_encoder_iterator.go

```diff
@@ -140,16 +140,13 @@ func (eit *FloatEncoderAndIterator) readNextFloat(stream encoding.IStream) error
 		return nil
 	}
 
-	numLeadingZeros, err := stream.ReadBits(6)
+	numLeadingZeroesAndNumMeaningfulBits, err := stream.ReadBits(12)
 	if err != nil {
 		return err
 	}
 
-	numMeaningfulBits, err := stream.ReadBits(6)
-	if err != nil {
-		return err
-	}
-	numMeaningfulBits = numMeaningfulBits + 1
+	numLeadingZeros := (numLeadingZeroesAndNumMeaningfulBits & 4032) >> 6
+	numMeaningfulBits := (numLeadingZeroesAndNumMeaningfulBits & 63) + 1
```

Comment:
Can we make these two numbers consts? 

i.e. 
```go
const (
  bits12To6Set = 4032 // 1111 1100 0000
  bits6To0Set = 63 // 0011 1111
)
```

Reply:
sure, but replaced "set" with "mask"